src/DynamoCoreWpf/NodeViewCustomization/NodeViewCustomizations/FunctionNodeViewCustomization.cs

```diff
@@ -60,7 +62,10 @@ public void CustomizeView(Function function, NodeView nodeView)
             publishCustomNodeItem.Command = nodeView.ViewModel.DynamoViewModel.PublishSelectedNodesCommand;
             publishCustomNodeItem.CommandParameter = functionNodeModel;
 
-            nodeView.UpdateLayout();
+            nodeView.Dispatcher.BeginInvoke(new Action(() =>
+            {
+                nodeView.UpdateLayout();
+            }), DispatcherPriority.Background);
```

Comment:
I am not sure if this will make Graph Loading feel any faster.
Dispatcher.BeginInvoke will simply delay execution with the specified priority. So nodeView.UpdateLayout will still block the UI at some point 
@RobertGlobant20 did you manage to get any measurements on this part ? is it really any faster ? what exactly is the start and end point of Graph Loading - I assume since LoadGraph -> until all data is rendered and you can interact with Dynamo.

Reply:
@pinzart90 Please check out the time diff post by @RobertGlobant20 in the PR

Reply:
@pinzart90 i noticed that executing the UpdateLayout() inside the BeginInvoke is allowing to load the graph faster (probably due that is executed in a async call in background). 

Basically I decided to execute the UpdateLayout() async because when running dotTrace I noticed that `NodeView.UpdateLayout() ` was consuming too much time, I commented the line for the Python Editor node and I didn't see any difference at UI rendering level when playing with this node but honestly I don't know the impact of removing that line (at least I didn't see any difference).

In this image below I ran dotTrace with Dynamo executing UpdateLayout **without** the Dispatcher.BeginInvoke, as you can see UpdateLayout() is taking 43 755 ms and is the highest time 
![image](https://github.com/DynamoDS/Dynamo/assets/61755417/26cb452b-18ba-430e-88ad-7bb87030d224)

Reply:
This a 3 samples that I took with NodeView.UpdateLayout() executed in normal thread, as you can see the times are in the middle of after my fix and before my fix
![image](https://github.com/DynamoDS/Dynamo/assets/61755417/ef66aef5-b00f-491a-b88e-6a7bf77c2117)

Reply:
What I want to clarify is that running `Dispatcher.BeginInvoke(new Action(() => {}, DispatcherPriority.Background);` will not run the Action in another thread. All it means is that it will run it asynchronously at a later time (when the dispatcher starts to run  Background priority tasks). 
You are seeing better times in the `CustomizeView` function calls  because the `nodeView.UpdateLayout();` will not run then, it will run later coming from a lower level function like `Dispatcher.ProcessQueue() -> ... -> Dispatcher.ResumeExecution() ... -> nodeViewUpdateLayout().
THe user might see THe graph load up faster, but soon after the `nodeViewUpdateLayout` will kick in and it will block the UI thread until it finishes.

Reply:
I suggest you remove it and test the hell out of it (maybe at a bug bash). You can leave the code commented out with instructions as to why we removed it

Reply:
FYI @QilongTang @mjkkirschner

Reply:
I've removed the UpdateLayout() call for Python nodes.
https://github.com/DynamoDS/Dynamo/pull/14929/commits/2091c35ec1b38c955b7c69d1a83f829f537a1f8b